Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM node:16.13.0-slim as base
+FROM node:16.13.0-alpine as base
```

Comment:
i've historically hit random networking issues with alpine (musc etc), recommend we stick w/ slim?

Reply:
reverted